src/PostgREST/ApiRequest.hs

```diff
@@ -33,6 +33,8 @@ data Action = ActionCreate | ActionRead
 data Target = TargetIdent QualifiedIdentifier
             | TargetRoot
             | TargetUnknown [T.Text]
+-- | How to return the inserted data
+data PreferRepresentation = Full | Count | None deriving Eq
```

Comment:
Using the term `Count` here is confusing I think. There are three Prefer header settings the client can use: `return`, `count`, and `plurality`. Since the above line deals with `return` and not whether the count is calculated the name `LocationHeader` would probably be a better choice than `Count`.